app/models/office/event/organization_changed.rb

```diff
@@ -0,0 +1,9 @@
+module Office
```

Comment:
The namespace should be `Laboratory`, not `Office`, since nuntius creates - or at least, should create - an event handler in the consumer application namespace, not the producer application namespace.